### ISSUE TITLE
broken download link - srtmdem

### ISSUE THREAD

**Issue by Ohm-Np** [mapme-initiative/mapme.biodiversity#73]
The link to download srtm dem urls in the package seem to be broken. However, downloading the srtm tiles manually using the same link through PC works.

```r
<simpleError in open.connection(x, "rb"): HTTP error 403.>
<simpleWarning: Download for resource srtmdem failed. Returning unmodified portfolio object.>
Warning message:
Download for resource srtmdem failed. Returning unmodified portfolio object. 
```
[Link to source](https://srtm.csi.cgiar.org/srtmdata/)

**Comment by Ohm-Np** [comment]
looks like we cannot scrape the web-element from this url: 
`https://srtm.csi.cgiar.org/wp-content/uploads/files/srtm_5x5/TIFF/`

This is the message I got when I try to open this base-url:
![Capture](https://user-images.githubusercontent.com/56826085/174810182-04cef513-bfdf-48b0-ae3d-690c7dff1c80.JPG)

**Comment by goergen95** [comment]
Yeah, we need an alternative here. The download of zip files still works but we cannot list all available tiles. A workaround could be to create a tileindex of SRTM and keep it in the package. However, I've found the website not really reliable so we might be better off looking for an alternative source (e.g. [earthdata](https://www.earthdata.nasa.gov/))...

**Comment by goergen95** [comment]
fixed via [8acb7ff](https://github.com/mapme-initiative/mapme.biodiversity/commit/8acb7ff6fb87793752ba7f832566b2f253361a08) by reactivating the url existence check for SRTM data.
